packages/flutter_tools/lib/src/asset.dart

```diff
@@ -336,7 +336,7 @@ Map<_Asset, List<_Asset>> _parseAssets(
   if (manifestDescriptor == null)
     return result;
 
-  excludeDirs = excludeDirs.map(
+  excludeDirs = excludeDirs.map<String>(
     (String exclude) => fs.path.absolute(exclude) + fs.path.separator).toList();
```

Comment:
nit: this line is wrapped weird. It's not clear where the function arguments end. Maybe put a newline before the penultimate ) ?